sdk/communication/azure-communication-chat/swagger/README.md

```diff
@@ -34,7 +34,7 @@ To update generated files for chat service, run the following command
 
 ### Code generation settings
 ``` yaml
-input-file: swagger.json
```

Comment:
Let's remove the instruction on top of the README to run the swagger_replace.ps1 script